resources/src/main/res/values/strings.xml

```diff
@@ -495,4 +495,17 @@
   <string name="widget_linear_layout_horz">LinearLayout (H)</string>
   <string name="widget_linear_layout_vert">LinearLayout (V)</string>
   <string name="widget_relative_layout">RelativeLayout</string>
+
+  <!-- GIT -->
+  <string name="cancel">Cancel</string>
+  <string name="okay">Okay</string>
```

Comment:
These strings are not needed as they are already available as `android.R.string.ok` and `android.R.string.cancel`.

Reply:
Okay